### ISSUE TITLE
Sort by "Oldest first" in Following only returns max 3 months old claims, instead of the acutally oldest ones

### ISSUE THREAD

**Issue by keikari** [lbryio/lbry-desktop#7733]
`claim_search` in Following page has `release_time: ">3 months ago"`. Which makes sort by "oldest first" not return oldest claims.  
Also when sorting by "newest first", can only find claims up to that time.  

![oldest-first](https://user-images.githubusercontent.com/34790748/201656758-95081457-e956-46ea-a7b5-0c5e8e8704f6.png)

https://www.reddit.com/r/lbry/comments/yuewmm/lbry_desktop_client_sorting_by_oldest_first/

**Comment by dgarrett01** [comment]
This can be fixed by commenting out or removing these if conditions in ui\component\claimListDiscover\view.jsx...

if (
        (options.channel_ids && options.channel_ids.length > 20) ||
        (options.any_tags && options.any_tags.length > 20)
      ) {
        options.release_time = `>${Math.floor(moment().subtract(3, CS.FRESH_MONTH).startOf('week').unix())}`;
      } else if (
        (options.channel_ids && options.channel_ids.length > 10) ||
        (options.any_tags && options.any_tags.length > 10)
      ) {
        options.release_time = `>${Math.floor(moment().subtract(1, CS.FRESH_YEAR).startOf('week').unix())}`;
      } else {
        // Hack for at least the New page until https://github.com/lbryio/lbry-sdk/issues/2591 is fixed
        options.release_time = `<${Math.floor(moment().startOf('minute').unix())}`;
      }

**Comment by jessopb** [comment]
Yeah, I'll release a test to make sure we can greatly reduce or eliminate these.
